cpp/velox/jni/VeloxJniWrapper.cc

```diff
@@ -84,6 +82,27 @@ JNIEXPORT void JNICALL Java_io_glutenproject_init_BackendJniWrapper_initializeBa
   JNI_METHOD_END()
 }
 
+JNIEXPORT jlong JNICALL Java_io_glutenproject_init_BackendJniWrapper_createExecutionCtx( // NOLINT
+    JNIEnv* env,
+    jclass) {
+  JNI_METHOD_START
+  auto executionCtx = gluten::createExecutionCtx();
+  return reinterpret_cast<jlong>(executionCtx);
+  JNI_METHOD_END(-1)
+}
+
+JNIEXPORT void JNICALL Java_io_glutenproject_init_BackendJniWrapper_releaseExecutionCtx( // NOLINT
+    JNIEnv* env,
+    jclass,
+    jlong ctxHandle) {
+  JNI_METHOD_START
+  auto executionCtx = reinterpret_cast<gluten::ExecutionCtx*>(ctxHandle);
+  GLUTEN_CHECK(executionCtx != nullptr, "ExecutionCtx should not be null.");
+
+  gluten::releaseExecutionCtx(executionCtx);
+  JNI_METHOD_END()
```

Comment:
Any reason putting the functions to `VeloxJniWrapper.cc` ? Can we use core module ?

Reply:
Just follow `InitializeBackend`, I'm ok for put these functions to `core` or `velox` dir.

Reply:
OK. Then let's open a new PR later on to do the clean-up.

Ideally `gluten-data` doesn't call `cpp/velox` directly. We should finally avoid that.